pkg/csi/cinder/controllerserver.go

```diff
@@ -132,6 +137,10 @@ func (cs *controllerServer) ControllerPublishVolume(ctx context.Context, req *cs
 
 	err = cs.Cloud.WaitDiskAttached(instanceID, volumeID)
 	if err != nil {
+		if cpoerrors.IsNotFound(err) {
+			klog.V(4).Infof("Disk %s is already deleted.", volumeID)
```

Comment:
:nit: we could use log level 3 for consistency here?

Reply:
I think we might need to do this check after AttachVolume to check for NOtFound and return accordingly?

Reply:
already done in sanity fix